maintainers/maintainer-list.nix

```diff
@@ -8417,6 +8417,12 @@
     githubId = 997855;
     name = "Narazaki Shuji";
   };
+  shofius = {
```

Comment:
Could you please do the adding to `maintainer-list.nix ` in a separate commit?

Reply:
Should that be a separate commit in the same PR, or should I make a new PR for that?

Reply:
Same PR, just a separate commit.